### PR TITLE
Added color statements in echo statement

### PR DESCRIPTION
**Pull request by rchikatw**

### CODE REVIEW

File: utils.sh [hunk 1]
```diff
@@ -40,10 +47,10 @@ validate() {
   for var in "$@"
   do
     if hash $var 2>/dev/null; then
-        echo "OK, you have $var installed. We will use that."
+        echo "${Green}OK, you have $var installed. We will use that."
```
Comment (rewantsoni) [inline utils.sh]: I think we can remove this, WDYT?

File: utils.sh [hunk 2]
```diff
@@ -27,7 +34,7 @@ storeKubeconfigAndLoginCluster() {
   response=$(ocm get /api/clusters_mgmt/v1/clusters/${1}/credentials 2>&1)
   kind=$(echo $response | jq .kind | sed "s/\"//g")
   if [[ $kind == "ClusterCredentials" ]]; then
-    echo "Cluster id found, getting kubeconfig"
+    echo "${Green}Cluster id found, getting kubeconfig"
```
Comment (rewantsoni) [inline utils.sh]:
```suggestion
    echo "${Green}Cluster ID found, getting kubeconfig"
```

File: updateEBSVolumeTags.sh [hunk 3]
```diff
@@ -26,12 +26,12 @@ if [[ "${1}" == "-h" ]] || [[ "${1}" == "--help" ]]; then
   exit 0
 fi
 
-echo "Reading volume IDs from backup"
+echo "${EndColor}Update EBS volume tags started${EndColor}"
```
Comment (rewantsoni) [inline updateEBSVolumeTags.sh]:
```suggestion
echo "${Green}Update EBS volume tags started${EndColor}"
```

File: restoreProvider.sh [hunk 4]
```diff
@@ -408,13 +406,13 @@ checkMonStatus
...
 
-echo -e "\nScaling up the rook ceph operator"
+echo -e "\n${Cyan}Scaling up the rook ceph operator${EndColor}"
 kubectl scale deployment rook-ceph-operator --replicas 1
 
 sleep 60
 checkDeployerCSV
 
-echo -e "\nRestart the rook ceph tools pod"
+echo -e "\n${Cyan}Restart the rook ceph tools pod${EndColor}"
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
echo -e "\n${Cyan}Restarting the rook ceph tools pod${EndColor}"
```

File: restoreProvider.sh [hunk 5]
```diff
@@ -302,11 +298,11 @@ applyOsds() {
 applyStorageConsumers() {
 
   # Get the names of all the storage consumer files in the backup directory and apply them
-  echo -e "\nApplying Storage consumers"
+  echo -e "\n${Cyan}Applying Storage consumers${EndColor}"
   consumers=`ls  $backupDirectoryName/storageconsumers`
   for entry in $consumers
   do
-    echo "applying Consumer with name: "$entry
+    echo "${Cyan}Applying Consumer with name: ${EndColor}"$entry
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
    echo "${Cyan}Applying StorageConsumer: ${EndColor}"$entry
```

File: restoreProvider.sh [hunk 6]
```diff
@@ -288,7 +284,7 @@ checkMonStatus() {
 applyOsds() {
 
   # Iterate over the OSD deployment files
-  echo -e "\nApplying the deployments for osds"
+  echo -e "\n${Cyan}Applying the deployments for osds${EndColor}"
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
  echo -e "\n${Cyan}Applying the deployments for OSD's${EndColor}"
```

File: restoreProvider.sh [hunk 7]
```diff
@@ -268,14 +264,14 @@ updateConfigMap() {
...
-  echo -e "\nWaiting for all the mon's pod to come in Running status and expected mon container count to be available"
+  echo -e "\n${Cyan}Waiting for all the mon's pod to come in Running status and expected mon container count to be available${EndColor}"
   for mon in "${mons[@]}";
   do
     while true
     do
       podStatus=$(kubectl get pods | grep rook-ceph-mon-${mon} | awk '{ print $3; exit }')
       containerCount=$(kubectl get pods | grep rook-ceph-mon-${mon} | awk '{ print $2; exit }')
-      echo "for mon "${mon}" podStatus is "$podStatus" containerCount is "$containerCount
+      echo "${Blue}for mon ${EndColor}"${mon}"${Blue} podStatus is ${EndColor}"$podStatus"${Blue} containerCount is ${EndColor}"$containerCount
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
      echo "${Blue}Mon deployment ${EndColor}"${mon}"${Blue} podStatus is ${EndColor}"$podStatus"${Blue} containerCount is ${EndColor}"$containerCount
```

File: restoreProvider.sh [hunk 8]
```diff
@@ -237,7 +233,7 @@ injectMonMap() {
 
     sleep 5
 
-    echo -e "\napplying rook ceph mon deployment"
+    echo -e "\n${Cyan}applying rook ceph mon deployment${EndColor}"
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
    echo -e "\n${Cyan}Applying deployments for rook ceph mon${EndColor}"
```

File: restoreProvider.sh [hunk 9]
```diff
@@ -220,12 +217,11 @@ injectMonMap() {
     echo "extractMonmap: "${extractMonmap}
     echo "injectMonmap: "${injectMonmap}
 
-    echo -e "\nWaiting for mon "$monName" pod to come in Running status"
     while true
     do
       podStatus=$(kubectl get pods | grep $mon | awk '{ print $3; exit }')
       containerCount=$(kubectl get pods | grep $mon | awk '{ print $2; exit }')
-      echo "podStatus is "$podStatus
+      echo "${Blue}Waiting for mon ${EndColor}"$monName"${Blue} pod to come in Running status and podStatus is ${EndColor}"$podStatus
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
      echo "${Blue}Waiting for mon ${EndColor}"$monName"${Blue} pod to come in Running status, current podStatus is ${EndColor}"$podStatus
```

File: restoreProvider.sh [hunk 10]
```diff
@@ -183,27 +181,26 @@ injectMonMap() {
...
     sleep 2
     cat <<< $(jq 'del(.spec .template  .spec .containers[0] .args[]|select(. | contains("--public-addr")))' ${mon}.json) > ${mon}.json
     cat <<< $(jq --arg publicIp $publicIp '.spec .template  .spec .containers[0] .args +=[$publicIp]' ${mon}.json ) > ${mon}.json
 
-    echo -e "\nWaiting for mon "$monName" pod to come in crashbackloop status"
     while true
     do
       podStatus=$(kubectl get pods | grep $mon | awk '{ print $3; exit }')
-      echo "podStatus is "$podStatus
+      echo "${Blue}Waiting for mon ${EndColor}"$monName"${Blue} pod to come in crashbackloop status and podStatus is ${EndColor}"$podStatus
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
      echo "${Blue}Waiting for mon ${EndColor}"$monName"${Blue} pod to come in CrashLoopBackOff status, current podStatus is ${EndColor}"$podStatus
```

File: restoreProvider.sh [hunk 11]
```diff
@@ -136,8 +135,7 @@ applySecrets() {
 
 prepareData() {
 
-  echo -e "\nPreparing data"
-  echo -e "\nMapping monName to nodeName and nodeIP"
+  echo -e "\n${Cyan}Preparing data and Mapping monName to nodeName and nodeIP${EndColor}"
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
  echo -e "\n${Cyan}Mapping mon's to nodeName and nodeIP${EndColor}"
```

File: restoreProvider.sh [hunk 12]
```diff
@@ -123,7 +122,7 @@ applyPersistentVolumeClaims() {
 applySecrets() {
 
   # Apply the secrets
-  echo -e "\nApplying required secrets for rook-ceph"
+  echo -e "\nA${Cyan}pplying required secrets for rook-ceph${EndColor}"
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
  echo -e "\nA${Cyan}pplying required secrets for rook-ceph cluster${EndColor}"
```

File: restoreProvider.sh [hunk 13]
```diff
@@ -38,26 +38,25 @@ declare -A mons
...
 
 }
 
 checkDeployerCSV() {
-  echo -e "\nWaiting for managed-fusion to come in Succeeded phase"
   while true
   do
     csvStatus=$(kubectl get csv | grep managed-fusion | awk '{ print $6; exit }')
-    echo "csvStatus is "$csvStatus
+    echo "${Blue}Waiting for managed-fusion to come in Succeeded phase and csvStatus is ${EndColor}"$csvStatus
```
Comment (rewantsoni) [inline restoreProvider.sh]:
```suggestion
    echo "${Blue}Waiting for managed-fusion CSV to come in Succeeded phase, current csvStatus is ${EndColor}"$csvStatus
```

File: migrateConsumer.sh [hunk 14]
```diff
@@ -342,3 +341,5 @@ patchFSPV
 patchRBDPVC
 
 patchFSPVC
+
+echo "${Green}Migrate consumer script completed!${EndColor}"
```
Comment (rewantsoni) [inline migrateConsumer.sh]: Add a new line at the end of the file

File: migrateConsumer.sh [hunk 15]
```diff
@@ -197,8 +195,8 @@ checkStorageClassClaim() {
     claimStatusTwo=$(kubectl get storageclassclaim -n ${operatorNamespace} --no-headers | awk '{print $5}' | awk 'FNR == 2')
     claimNameTwo=$(kubectl get storageclassclaim -n ${operatorNamespace} --no-headers | awk '{print $1}' | awk 'FNR == 2')
 
-    echo "StorageClassClaim "${claimNameOne}" Stauts is "$claimStatusOne
-    echo "StorageClassClaim "${claimNameTwo}" Stauts is "$claimStatusTwo
+    echo "${Blue}Waiting for storageClassClaims to come in Connected phase, StorageClassClaim ${EndColor}"${claimNameOne}"${Blue} Stauts is ${EndColor}"$claimStatusOne
+    echo "${Blue}Waiting for storageClassClaims to come in Connected phase, StorageClassClaim ${EndColor}"${claimNameTwo}"${Blue} Stauts is ${EndColor}"$claimStatusTwo
```
Comment (rewantsoni) [inline migrateConsumer.sh]:
```suggestion
    echo "${Blue}Waiting for storageClassClaim ${EndColor}"${claimNameOne}" ${Blue}to come to Ready phase, current phase is ${EndColor}"$claimStatusOne
    echo "${Blue}Waiting for storageClassClaim ${EndColor}"${claimNameTwo}" ${Blue}to come to Ready phase, current phase is ${EndColor}"$claimStatusTwo
```

File: migrateConsumer.sh [hunk 16]
```diff
@@ -169,11 +169,10 @@ createStorageClient() {
 }
 
 checkStorageClient() {
-  echo -e "\nWaiting for storageclient to come in Connected phase"
   while true
   do
     clientStatus=$(kubectl get storageclient ${storageClientName} -n ${operatorNamespace} --no-headers | awk '{ print $2; exit }')
-    echo "client Stauts is "$clientStatus
+    echo "${Blue}Waiting for storageclient to come in Connected phase and client Stauts is ${EndColor}"$clientStatus
```
Comment (rewantsoni) [inline migrateConsumer.sh]:
```suggestion
    echo "${Blue}Waiting for storageclient to come in Connected phase, current Stauts is ${EndColor}"$clientStatus
```

File: migrateConsumer.sh [hunk 17]
```diff
@@ -143,13 +143,13 @@ spec:
...
 }
-
 checkOperatorCSV() {
-  echo -e "\nWaiting for ocs-client-operator to come in Succeeded phase"
+  
   while true
   do
     csvStatus=$(kubectl get csv -n ${operatorNamespace} | grep ocs-client-operator | awk '{ print $7; exit }')
-    echo "csvStatus is "$csvStatus
+    echo -e "\n${Blue}Waiting for ocs-client-operator to come in Succeeded phase and csvStatus is ${EndColor}"$csvStatus
```
Comment (rewantsoni) [inline migrateConsumer.sh]:
```suggestion
    echo -e "\n${Blue}Waiting for ocs-client-operator CSV to come to Succeeded phase, current CSV Status is ${EndColor}"$csvStatus
```

File: migrateConsumer.sh [hunk 18]
```diff
@@ -74,7 +74,7 @@ releasePV() {
           kubectl delete pv ${pvName}
           break
       fi
-      echo "waiting for PV "$pvName" to go in released state, current state is "$pvStatus
+      echo "${Blue}waiting for PV "$pvName" to go in released state, current state is ${EndColor}"$pvStatus
```
Comment (rewantsoni) [inline migrateConsumer.sh]:
```suggestion
      echo "${Blue}waiting for PV "$pvName" to go to released state, current state is ${EndColor}"$pvStatus
```

File: migrate.sh [hunk 19]
```diff
@@ -142,7 +142,7 @@ kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}'
 kubectl get storagesystem -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl patch storagesystem -p '{"metadata":{"finalizers":null}}' --type=merge -n openshift-storage
 kubectl get storagecluster -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl patch storagecluster -p '{"metadata":{"finalizers":null}}' --type=merge -n openshift-storage
 
-echo -e "\nDeletion of Old Service cluster is started, It will take some to delete the service."
+echo -e "\n${Green}Deletion of Old Service cluster is started, It will take some to delete the service.${EndColor}"
 
 cleanup
-echo -e "\nMigration Process completed!"
+echo -e "\n${Green}Migration Process completed!${EndColor}"
```
Comment (rewantsoni) [inline migrate.sh]: Add a new line

File: migrate.sh [hunk 20]
```diff
@@ -142,7 +142,7 @@ kubectl get storageconsumer -n openshift-storage --no-headers | awk '{print $1}'
 kubectl get storagesystem -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl patch storagesystem -p '{"metadata":{"finalizers":null}}' --type=merge -n openshift-storage
 kubectl get storagecluster -n openshift-storage --no-headers | awk '{print $1}' | xargs kubectl patch storagecluster -p '{"metadata":{"finalizers":null}}' --type=merge -n openshift-storage
 
-echo -e "\nDeletion of Old Service cluster is started, It will take some to delete the service."
+echo -e "\n${Green}Deletion of Old Service cluster is started, It will take some to delete the service.${EndColor}"
```
Comment (rewantsoni) [inline migrate.sh]:
```suggestion
echo -e "\n${Green}Deletion of Old Provider Service cluster is started, the service will be deleted soon.${EndColor}"
```

File: migrate.sh [hunk 21]
```diff
@@ -61,33 +61,33 @@ do
...
+echo -e "\n${BoldCyan}Enter the clusterID of the new cluster:${EndColor}"
 read restoreClusterID
 
 loginCluster $1 "$restoreClusterID"
 
-echo -e "\nMigrating to the new Provider"
+echo -e "\n${Cyan}Migrating to the new Provider${EndColor}"
 sh ./restoreProvider.sh
-echo "Migration of Provider Complete!"
+echo "${Green} Migration of provider is completed!${EndColor}"
```
Comment (rewantsoni) [inline migrate.sh]: We can move these echo inside the restoreProvider Script like we did for other scripts

File: migrate.sh [hunk 22]
```diff
@@ -61,33 +61,33 @@ do
...
-echo -e "\nTaking a backup of resouces required for backup"
+echo -e "\n${Cyan}Taking a backup of resouces required for backup${EndColor}"
 sh ./backupResources.sh
 echo "Backup Complete!"
 
-echo -e "\nScaling down the rook-ceph pods connected to the EBS Volumes"
+echo -e "\n${Cyan}Scaling down the rook-ceph pods connected to the EBS Volumes${EndColor}"
 sh ./freeEBSVolumes.sh
-echo "Scaling down Complete!"
+echo "${Cyan}Scaling down Complete!${EndColor}"
```
Comment (rewantsoni) [inline migrate.sh]: We can move these echo inside the freeEBSVolume Script like we did for other scripts

File: migrate.sh [hunk 23]
```diff
@@ -61,33 +61,33 @@ do
...
   fi
 done
 
-echo "Validation Complete!"
+echo "${Cyan}Validation Complete!${EndColor}"
 
 loginCluster $1 "$backupClusterID"
 
-echo -e "\nTaking a backup of resouces required for backup"
+echo -e "\n${Cyan}Taking a backup of resouces required for backup${EndColor}"
```
Comment (rewantsoni) [inline migrate.sh]: We can move these echo inside the backupProvider Script like we did for other scripts

File: migrate.sh [hunk 24]
```diff
@@ -61,33 +61,33 @@ do
...
 sh ./backupResources.sh
 echo "Backup Complete!"
```
Comment (rewantsoni) [inline migrate.sh]: We can move these echo inside the backupProvider Script like we did for other scripts

File: migrate.sh [hunk 25]
```diff
@@ -61,33 +61,33 @@ do
     boundPVs[${#boundPVs[@]}]=${volumeAttachment}
   done
   if [[ "${#boundPVs[@]}" > "0" ]]; then
-    echo -e "\nThere are still PVC which are using by pods please scale down the application pod and re-run the script\n"
-    echo -e "Cluster ID: "$consumerClusterID
+    echo -e "\n${Red}There are still PVC which are using by pods please scale down the application pod and re-run the script\n${EndColor}"
+    echo -e "${Red}Cluster ID: ${EndColor}"$consumerClusterID
```
Comment (rewantsoni) [inline migrate.sh]:
```suggestion
    echo -e "\n${Red}On Consumer with ClusterID: ${EndColor}"$consumerClusterID"${Red} We still have some applications using the PVC, Please scale down/delete the pods and re-run the script.\nPVC's being used are: ${EndColor}"
```

File: freeEBSVolumes.sh [hunk 26]
```diff
@@ -22,6 +22,8 @@ if [[ "${1}" == "-h" ]] || [[ "${1}" == "--help" ]]; then
   exit 0
 fi
 
+echo "${Green}Free EBS script started${EndColor}"
```
Comment (rewantsoni) [inline freeEBSVolumes.sh]: Maybe have a different log message here?

**Review by rewantsoni — CHANGES_REQUESTED** [review]

File: utils.sh (hunk 1 above)
Comment (rchikatw) [reply utils.sh]: Yes we can.

**Review by rchikatw — COMMENTED** [review]
I gone through all of the comments all looks valid so resolving all in next commit

File: utils.sh [hunk 27]
```diff
@@ -42,8 +49,8 @@ validate() {
     if hash $var 2>/dev/null; then
         echo "OK, you have $var installed. We will use that."
```
Comment (rewantsoni) [inline utils.sh:50]: Can we remove this?

**Review by rewantsoni — CHANGES_REQUESTED** [review]

**Review by rewantsoni — APPROVED** [review]

**Pull request merged**